extensions/sample/.classpath

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
-	<classpathentry kind="output" path="module/MOD-INF/classes"/>
-</classpath>
```

Comment:
Same comment.

Reply:
sample project is not necessary

Reply:
I think the sample extension is definitely useful - I don't see why it would be a good thing to delete its `.classpath`. Again this is unrelated to data packages so let's not delete this file in this PR.

Reply:
Again, this already covered by OpenRefine project. No need redundancy to setup the project